group_vars/osds.yml.sample

```diff
@@ -115,22 +116,33 @@ dummy:
 # /dev/sda: PTTYPE="gpt"
 # /dev/sda1: UUID="9c43e346-dd6e-431f-92d8-cbed4ccb25f6" TYPE="xfs" PARTLABEL="ceph data" PARTUUID="749c71c9-ed8f-4930-82a7-a48a3bcdb1c7"
 # /dev/sda2: PARTLABEL="ceph block" PARTUUID="e6ca3e1d-4702-4569-abfa-e285de328e9d"
-#journal_collocation: false
+#
+
+#osd_scenario: dummy
+#valid_osd_scenarios:
+#  - collocated
+#  - non-collocated
 
 
-# II.  Second scenario:  N journal devices for N OSDs
+# II.  Second scenario:  non-collocated
+#
+# To enable this scenario do: osd_scenario: collocated
+#
 # Use 'true' for 'raw_multi_journal' to enable this scenario
 # List devices under 'devices' variable above and
-# write journal devices for those under 'raw_journal_devices'
+# write journal devices for those under 'dedicated_devices'
```

Comment:
How do you feel about renaming ``dedicated_devices`` to ``journal_devices``? I think ``journal`` needs to be in the name of the variable somehow. It's a bit confusing to have to define ``devices`` and ``dedicated_devices``, the difference is not very apparent.